README.md

```diff
@@ -6,40 +6,40 @@ User application for Agoric Inter Protocol--Vaults, BLD Boost, Liquidations, etc
 
 1. Download and build the latest copy of `agoric-sdk`.
 
-```
-cd agoric-sdk
-yarn && yarn build
-```
+   ```
+   cd agoric-sdk
+   yarn && yarn build
+   ```
```

Comment:
backticks or indentation. to have both will render the backticks in the codeblock.

But I think we're better off keeping them because backticks can indicate the syntax to highlight and indentation can't.
```suggestion
```sh
cd agoric-sdk
yarn && yarn build
```

Reply:
I believe you are incorrect, as visible in the rendering at [`gibson-2023-03-readme`](https://github.com/Agoric/dapp-inter/blob/gibson-2023-03-readme/README.md) . This indentation ensures that the code block is a child of the preceding list item (cf. [CommonMark Spec: List items](https://spec.commonmark.org/0.30/#list-items)) rather than interrupting the list as visible at [`main`](https://github.com/Agoric/dapp-inter/blob/main/README.md) .

Reply:
The rendering I looked at was in VS Code's Markdown preview so maybe so